sled-agent/src/instance.rs

```diff
@@ -130,6 +131,10 @@ pub enum Error {
     Terminating,
 }
 
+// Make this less gross.
```

Comment:
I might delete this comment, unless you have something more actionable in mind -- this is pretty standard for progenitor error types, right?

Reply:
Agreed; progenitor is great.